plugins/kubectl-casskop

```diff
@@ -17,6 +17,7 @@
 import argparse
```

Comment:
Rebase your code on master. You should not have the content of my merged branch here.

Reply:
it has been merged so that won't be a problem